File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;


class UberVan extends Car{
     Map<String, Map<String, Integer>> typeCarAccepted;
     ArrayList<String> seatsMaterial; 

    public UberVan(String license, Account driver) {
            super(license, driver);
    }

    // public UberVan2(String license, Account driver, Map<String, Map<String, Integer>> typeCarAccepted,
    //                 ArrayList<String> seatsMaterial) {
    //         super(license, driver);
    //         this.typeCarAccepted = typeCarAccepted;
    //         this.seatsMaterial = seatsMaterial;
    // }

      
    public void setPassenger(int passengers) {
        if(passengers == 6){
            this.passengers = passengers;
        }else {
            System.out.println("Necesitas asignar 6 pasajeros");
        }
    }
    
    }

File: PHP/UberVan.php
<?php
require_once('Car.php');
    class UberBlack extends Car {
        public $typeCarAccepted;
        public $seatMaterial;

        public function _construct($license, $driver, $typeCarAccepted, $seatMaterial) {
            parent::_construct($license,$driver);
            $this->typeCarAccepted = $typeCarAccepted;
            $this->seatsMaterial = $seatMaterial;
        }

    }

        public function setPassenger($passenger) {
            if($passenger == 6) {
                $this->passenger = $passenger;
            } else {
                echo "Necesitas asignar 6 pasajeros";
            }
        }

?>